### PR TITLE
EDUCATOR-5387: Add Messaging for why staff can not create an ORA grade override

### PR DESCRIPTION
**Pull request by jansenk**
**TL;DR -** rather than hiding the staff override section of the staff area, show the section but only with an error message

**What changed?**

- When grades are frozen or the submission is cancelled, still show the "Submit Assessment/Team Grade Override" dropdown section, but display a warning message rather than the override form.

**Grades are frozen**
Screen Shot 2020-10-26 at 2 35 03 PM

**The response is cancelled**
cancelled

**The response is cancelled and grades are also frozen**
Screen Shot 2020-10-26 at 2 36 24 PM

**Testing tips**
- Grades are considered "frozen" if it is over 30 days past the end of the course, so if you go into studio and change the course end date to more than 30 days in the past, grades will be considered frozen

**Developer Checklist**
- [x] Reviewed the [release process](./release_process.md)
- [x] Translations up to date
- [x] JS minified, SASS compiled
- [x] Test suites passing on Jenkins
- [ ] Bumped version number in [setup.py](../setup.py) and [package.json](../package.json)
- [x] Copy okayed by sapana

JIRA: [EDUCATOR-5387](https://openedx.atlassian.net/browse/EDUCATOR-5387)

FIY: @edx/masters-devs-gta

### CODE REVIEW

**Comment by jansenk** [comment]
Translations are in their own commit for reviewers' sake

**Comment by jansenk** [comment]
Updated message text for frozen grades

File: openassessment/templates/openassessmentblock/staff_area/oa_student_info.html [hunk 1]
```diff
@@ -184,29 +184,44 @@ <h2 class="staff-info__title ui-staff__subtitle">
...
+                        <h5 class="message__title">{% trans "Unable to perform grade override" %}</h5>
+                        {% if are_grades_frozen %}
+                            <div class="message__content">
+                                {% trans "Grades are frozen. Grades are automatically frozen 30 days after the course end date." %}
+                            </div>
+                        {% endif %}
+                        <br>
+                        {% if workflow_cancellation %}
+                            <div class="message__content">
+                                {% trans "This submission has been cancelled and cannot recieve a grade. Refer to documentation for how to reset a cancelled submission." %}
```
Comment (Andytr1) [inline openassessment/templates/openassessmentblock/staff_area/oa_student_info.html:213]: I think the last sentence (Refer to documentation...) is redundant since we aren't saying where the documentation is.

**Review by Andytr1 — APPROVED** [review]
lgtm

**Review by nsprenkle — COMMENTED** [review]
LGTM, will test locally pre-approval.

File: openassessment/templates/openassessmentblock/staff_area/oa_student_info.html [hunk 2]
```diff
@@ -184,29 +184,44 @@ <h2 class="staff-info__title ui-staff__subtitle">
...
+            <div class="wrapper--input">
+                {% if are_grades_frozen or workflow_cancellation %}
+                    <div class="step__message message message--warning" tabindex="-1">
+                        <h5 class="message__title">{% trans "Unable to perform grade override" %}</h5>
+                        {% if are_grades_frozen %}
+                            <div class="message__content">
+                                {% trans "Grades are frozen. Grades are automatically frozen 30 days after the course end date." %}
+                            </div>
+                        {% endif %}
+                        <br>
```
Comment (nsprenkle) [inline openassessment/templates/openassessmentblock/staff_area/oa_student_info.html:210]: Minor display nit: this appears as extra padding after the "frozen" case or before the "cancelled" case when either exists alone. Can probably fix by removing the `<br>` and doing the spacing in CSS.

**Review by nsprenkle — APPROVED** [review]

File: openassessment/templates/openassessmentblock/staff_area/oa_student_info.html (hunk 2 above)
Comment (jansenk) [reply openassessment/templates/openassessmentblock/staff_area/oa_student_info.html]: Screen Shot 2020-10-27 at 11 16 00 AM

How's that?

File: openassessment/templates/openassessmentblock/staff_area/oa_student_info.html (hunk 1 above)
Comment (jansenk) [reply openassessment/templates/openassessmentblock/staff_area/oa_student_info.html:213]: Sapana okayed the text but I'll bring this up to her

File: openassessment/templates/openassessmentblock/staff_area/oa_student_info.html (hunk 2 above)
Comment (nsprenkle) [reply openassessment/templates/openassessmentblock/staff_area/oa_student_info.html]: Fab :shipit:
